### PR TITLE
Implemented FigureSupplier class who generate random figure with rand…

### PR DESCRIPTION
**Pull request by DrTymkivstain**
…om properties, 5 classes of figures, enum Colors with colors, 2 interfaces

### CODE REVIEW

File: src/main/java/core/basesyntax/Draw.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```
Comment (liliia-ponomarenko) [inline src/main/java/core/basesyntax/Draw.java:3]:
```suggestion
public interface Drawable  {
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier(Random random) {
+        this.random = random;
+    }
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        return String.valueOf(Colors.values()[index]);
```
Comment (liliia-ponomarenko) [inline src/main/java/core/basesyntax/ColorSupplier.java:14]: You can use method name() to get the name of the enum element

File: pom.xml [hunk 3]
```diff
@@ -19,6 +19,7 @@
     </properties>
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/junit/junit -->
```
Comment (liliia-ponomarenko) [inline pom.xml:22]:
```suggestion
```

File: src/main/java/core/basesyntax/Draw.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+    void name();
```
Comment (liliia-ponomarenko) [inline src/main/java/core/basesyntax/Draw.java:4]:
```suggestion
    void drawFigure();
```

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/ColorSupplier.java (hunk 2 above)
Comment (DrTymkivstain) [reply src/main/java/core/basesyntax/ColorSupplier.java:14]: Done?

**Pull request closed without merging**